packages/desktopjs-electron/src/electron.ts

```diff
@@ -359,8 +359,20 @@ export class ElectronContainer extends WebContainerBase {
 
             this.internalIpc = ipc || ((this.isRemote) ? require("electron").ipcRenderer : this.electron.ipcMain);
             this.ipc = this.createMessageBus();
+            this.nodeIntegration = null;
+            this.setOptions(options);
+        } catch (e) {
+            console.error(e);
+        }
 
-            if (!this.isRemote || (options && typeof options.isRemote !== "undefined" && !options.isRemote)) {
+        this.screen = new ElectronDisplayManager(this.electron);
+
+        this.globalShortcut = new ElectronGlobalShortcutManager(this.electron);
+    }
+
+    public setOptions(options: any) {
```

Comment:
Refactoring this out so it can be called at runtime vs only creation time is nice.